src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -275,6 +275,17 @@ private void readRcaEnabledFromConf() {
         });
   }
 
+  private void readAndUpdateMutesRcasDuringStart() {
+    try {
+      Set<String> rcasForMute = new HashSet<>(rcaConf.getMutedRcaList());
+      rcasForMute.forEach(mutedRca -> Stats.getInstance().addToMutedGraphNodes(mutedRca));
```

Comment:
I don't see that we are validating that there are Java classes with these names ?

Reply:
Let's add a test that exercises this path. I mean when during the start of RcaController

Reply:
> I don't see that we are validating that there are Java classes with these names ?

Done. Added check based on class name.

Reply:
> Let's add a test that exercises this path. I mean when during the start of RcaController

It is difficult to simulate this within a UT, since it relies on `lastModifiedTimeInMillisInMemory`, initialized at boot-up and `start()` being invoked. Will add an integration test to cover this.